resources/views/newDesign/company/show.blade.php

```diff
@@ -221,7 +221,7 @@
                 <span id="date-{{$comment->id}}">{{date('j.m.Y h:ia', strtotime($comment->updated_at))}}</span>
             </span>
             <p id="comment-{{$comment->id}}-description">{{$comment->comment}}</p>
-            @if(Auth::check() && Auth::id() == $comment->user_id)
+            @if(Auth::check() && Auth::id() == $comment->user_id && Auth::user()->isAdmin())
```

Comment:
Are you sure?
Only admin can remove comments and only their comments?

Reply:
yes, you are right. I've changed && to ||.